src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -81,20 +81,12 @@ export class StyleConfigurator {
       (style: StyleMap) => {
         style[mxgraph.mxConstants.STYLE_STARTARROW] = undefined;
         style[mxgraph.mxConstants.STYLE_ENDARROW] = undefined;
-        style[mxgraph.mxConstants.STYLE_EDGE] = undefined; // ensure no orthogonal segments, see also https://github.com/process-analytics/bpmn-visualization-js/issues/295
       },
     ],
     [
       AssociationDirectionKind.ONE,
       (style: StyleMap) => {
         style[mxgraph.mxConstants.STYLE_STARTARROW] = undefined;
-        style[mxgraph.mxConstants.STYLE_EDGE] = undefined; // ensure no orthogonal segments, see also https://github.com/process-analytics/bpmn-visualization-js/issues/295
-      },
-    ],
-    [
-      AssociationDirectionKind.BOTH,
-      (style: StyleMap) => {
-        style[mxgraph.mxConstants.STYLE_EDGE] = undefined; // ensure no orthogonal segments, see also https://github.com/process-analytics/bpmn-visualization-js/issues/295
       },
```

Comment:
ℹ️ If you prefer, we can keep the function with an empty implementation and a comment explaining that it is fully managed by defaults.

Reply:
Yes, to be sure that it is not an omission, I prefer to keep an empty function with comment ^^

Reply:
done in 56a8fc3